parsers/MN.py

```diff
@@ -143,6 +145,46 @@ def fetch_consumption(
     return consumption_list.to_list()
 
 
+def fetch_exchange(
+    zone_key1: ZoneKey,
+    zone_key2: ZoneKey,
+    session: Session = Session(),
+    target_datetime: datetime | None = None,
+    logger: Logger = getLogger(__name__),
+) -> list[dict]:
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates.")
+
+    sorted_zone_keys = ZoneKey("->".join(sorted([zone_key1, zone_key2])))
+
+    if sorted_zone_keys != "CN->MN":
+        raise NotImplementedError(
+            "This parser is only able to fetch CN->MN exchange data."
+        )
+
+    query_data = query(session, logger, ZoneKey("MN"))
+
+    russia_data = ru_fetch_exchange("MN", "RU-2", session, logger=logger)
+
+    exchange_list = ExchangeList(logger)
+
+    for data in russia_data:
+        if (
+            data["datetime"] == query_data["time"]
+            and data["sortedZoneKeys"] == "MN->RU-2"
+            and "netFlow" in data.keys()
+        ):
+            exchange_list.append(
+                datetime=query_data["time"],
+                zoneKey=sorted_zone_keys,
+                # We calculate the flow with the total imports and the MN->RU-2 exchange, as Mongolia has only two connections
+                netFlow=query_data["importMW"] + data["netFlow"],
+                source="https://ndc.energy.mn/",
```

Comment:
```suggestion
                netFlow=query_data["importMW"] - data["netFlow"],
```
The current implementation seems to be inversed.
Please double check this yourself.

It would also be good if we could add a test or two for this so we can validate the expected behavior.

Reply:
> seems to be inversed

I think it isn't.
Let's see:
CN->MN + RU->MN = Total Imports
CN->MN = Total Imports + MN->RU

I'll work on those tests. It would be optimal to be able to test the parser without requiring the sources to have matching data.

Reply:
the easiest way would probably be to add snapshot tests using mock data. Like I did in #6285.

That way it don't rely on an online source, changing data or creating manual tests are are both logic intensive and hard to maintain long term.

Reply:
I was thinking of doing something similar to what I did for the REE parser but I will take a look at these snapshot tests.